src/common/SConscript

```diff
@@ -1,6 +1,30 @@
 """Build common libraries"""
 import daos_build
 
+COMMON_SRC = ['debug.c', 'mem.c', 'fail_loc.c', 'lru.c',
+         'misc.c', 'pool_map.c', 'sort.c', 'btree.c', 'prop.c',
```

Comment:
(pylint-bad-continuation) Wrong continued indentation (add 5 spaces).

Reply:
the ' just needs to line up with prior line

Reply:
@jolivier23  I had to force push this pull request again as the previous version had many conflicts and was segfaulting. But I have incorporated all your comments in this revised version.